boards/riscv/qemu_riscv32e/doc/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _qemu_riscv32e:
```

Comment:
suggestion: move board addition to a separare commit

Reply:
done